javascript/javascript1/week1/ageify.js

```diff
@@ -0,0 +1,4 @@
+const yearOfBirth = 1987;
+const yearFuture = 2027 - yearOfBirth; 
```

Comment:
We'd like to have the future year in a variable of it's own, so

`const futureYear = 2027`